app/scripts/hglib.js

```diff
@@ -31,21 +31,42 @@ const launch = (element, config, options) => {
    * @param {Object} options Options that affect how the component is drawn and
    * and behaves.
    *
+   * @param {Object} componentCallback A callback to be called once the component
+   * is rendered. If it's not passed in, the block until the component is created.
+   *
+   * @param {Object} promise Return the ref as a promise or not.
```

Comment:
Where are those parameters? I only see `refAsPromise` but no `componentCallback` or `promise `

Reply:
Oh, I must have forgotten to take out componentCallback. I'll do that right now.